packages/neo-one-website/docs/0-installation/2-playground.md

```diff
@@ -25,6 +25,7 @@ This guide will walk you through getting started with the [NEO•ONE Playground]
 git clone https://github.com/neo-one-suite/neo-one-playground.git
 cd neo-one-playground
 yarn install
+# npm install
```

Comment:
I think it'd be better to put each of these above the `yarn` command - generally comments precede the article they're referring to.

Reply:
👍